packages/popover/test/a11y.test.js

```diff
@@ -26,8 +26,12 @@ describe('a11y', () => {
   });
 
   beforeEach(async () => {
-    popover = fixtureSync('<vaadin-popover></vaadin-popover>');
-    target = fixtureSync('<button>Target</button>');
+    [target, popover] = fixtureSync(`
```

Comment:
Modified the fixture to place popover after the target to have correct focus sequence. Placing popover before the target would not work properly. This is probably something worth documenting.